[Thorvg] Update Thorvg port to v0.13.0

rinechran opened the pull request:
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

rinechran committed:
[thorvg] update to 0.13.0

update : update line

LilyWangLL commented:
All features passed with following triplets:
```
x86-windows
x64-windows
x64-windows-static
```

rinechran committed:
update : Option modification

update : Option modification

update : baseline update

update : Option modification

update

rinechran commented:
![image](https://github.com/microsoft/vcpkg/assets/15512801/cc97f674-3bd5-45c9-9579-7efda43aad56)
Modified to meet the demands of the maintaine
